src/frame.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+const TOTAL_PINS = 10
+class Frame {
+  constructor() {
+    this.rolls = [];
+    this.score = 0;
```

Comment:
Probably better to put score into game as you want to know the score for the whole game rather than just the individual frame